exotica_core/include/exotica_core/kinematic_element.h

```diff
@@ -38,6 +38,17 @@
 
 namespace exotica
 {
+class VisualElement
+{
+public:
```

Comment:
Since this class contains fixed-size Eigen elements, it also needs EIGEN_MAKE_ALIGNED_ALLOCATOR_NEW (or similar)

Reply:
Couple other classes/structs elsewhere were missing this as well. Adding it now.